### ISSUE TITLE
[call history] colours for call history icons

### ISSUE THREAD

**Issue by DJCrashdummy** [UriahShaulMandel/BaldPhone#10]
i've got a little idea for a "nice-to-have", which could help to easier distinguish the 3 different kind of calls in the history.

simply colour the arrow-icons with the usual colours:
- **red** for **missed** calls
- **green** for **answered** (resp. "received") calls
- **blue** for **outgoing** calls (as neutral colour, because it is not 100% sure if you reached someone or not)

**Comment by UriahShaulMandel** [comment]
Great Idea! I'm on it.

**Comment by UriahShaulMandel** [comment]
I have changed your idea a bit, and I think it looks great!
what do you think?

<img width="200" alt="Screenshot_7" src="https://user-images.githubusercontent.com/43012268/55973658-bdbe7d80-5c8e-11e9-88e2-7c18743eff64.png">
<img width="198" alt="Screenshot_6" src="https://user-images.githubusercontent.com/43012268/55973661-bdbe7d80-5c8e-11e9-8a59-097ff7475c26.png">

**Comment by DJCrashdummy** [comment]
hmmm... interesting! :thinking: 

generally IMHO it looks good, but hopefully the conjunction between colour and call type is as clear as when just the icon were cloured...?
but for people with a weak vision it is clearly the better solution... an option to either colour the background or just the icons would perhaps be an overkill?

**Comment by DJCrashdummy** [comment]
hmmm... interesting! :thinking: 

generally IMHO it looks good, but hopefully the conjunction between colour and call type is as clear as when just the icon were cloured...?
but for people with a weak vision it is clearly the better solution... an option to either colour the background or just the icons would perhaps be an overkill?

**Comment by UriahShaulMandel** [comment]
I tried coloring only the icon, but it was a bit problematic for people with weak vision...
I think it's a bit overkill the have an option to either color the background or just the icons, but if you think I should I will add an option

**Comment by DJCrashdummy** [comment]
well, i also guess it would be a bit of an overkill...
if i were you, i wouldn't add the "coloured icons" option (for now), at least not until more users ask for it.

**Comment by UriahShaulMandel** [comment]
Great, and thanks again for helping!
